sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1218,7 +1260,9 @@ def _output(self, window, finished, state, watermark, maybe_ontime):
     if timestamp is None:
       # If no watermark hold was set, output at end of window.
       timestamp = window.max_timestamp()
-    else:
+    elif watermark >= window.end or not self.trigger_fn.has_ontime_pane():
+      # Hold the watermark in case there is an empty pane that needs to be fired
```

Comment:
This comment says to hold the watermark, but the code clears it. Reword? Or maybe negate and have the body be `pass` along with this comment.

Reply:
I negated then passed.